QUEST_LV_0200.tsv

```diff
@@ -5005,14 +5005,14 @@ QUEST_LV_0200_20150323_005004	You feel somewhat strange. You better open the sac
 QUEST_LV_0200_20150323_005005	{memo X}Talk
 QUEST_LV_0200_20150323_005006	{memo X}A person came out from the sack! You don't feel good, but you should talk to him.
 QUEST_LV_0200_20150323_005007	Try talking to him
-QUEST_LV_0200_20150323_005008	What happened? A person came out from the sack which Tadas told you to push out from the cliff. You should talk to him to find out why he was in the sack.
+QUEST_LV_0200_20150323_005008	What happened? A person came out from the sack which Tadas told you to push off the cliff. You should talk to him to find out why he was in the sack.
 QUEST_LV_0200_20150323_005009	{memo X}Bring potato rice cakes
-QUEST_LV_0200_20150323_005010	Auknews told you to find the food that he hid in the grass bushes. Seach through the grass bushes nearby.
+QUEST_LV_0200_20150323_005010	Yaunus told you to find the food that he hid in the grass bushes. Seach through the grass bushes nearby.
```

Comment:
It is, and I believe either end up being correct, but I can change it to 'J', since it really just sounds like a 'Y' when you say it.